docs/components/configMaps.md

```diff
@@ -12,30 +12,38 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
 
 # `gamepad.configMaps`
 
-This grade provides the configuration maps for the gamepad in the form of component's model data.
+// TODO: Add links referencing the "mix in" components' documentation.
```

Comment:
You'll need to use HTML comment syntax to avoid having the comment show up in the rendered output.